swift/ql/src/queries/Security/CWE-089/SqlInjection.qhelp

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+
+<p>
+If a database query (such as an SQL query) is built from user-provided data without sufficient sanitization, a user may be able to run malicious database queries.
```

Comment:
```suggestion
If a database query (such as a SQL query) is built from user-provided data without sufficient sanitization, a user may be able to run malicious database queries.
```

[MS Style guide treats as an acronym](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/sql-sql-server) ("a SQL...")

Reply:
Ah, I've seen that elsewhere and thought it was a mistake!  I've accepted your changes.